cxx-qt/src/lib.rs

```diff
@@ -16,13 +16,11 @@ use cxx_qt_gen::{extract_qobject, generate_qobject_rs};
 /// #[cxx_qt::bridge(namespace = "cxx_qt::my_object")]
 /// mod my_object {
 ///     #[derive(Default)]
-///     struct Data {
+///     struct RustObj {
```

Comment:
This is missing a #[cxx_qt::qobject] attribute

Reply:
Done